<!DOCTYPE html>
<html class="no-js" lang="en">

<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <title>Ruijia(Ray) Wang</title>
	<link rel="shortcut icon" href="favicon.ico" type="image/x-icon">
	<link rel="icon" href="favicon.ico" type="image/x-icon">
    <link href="https://fonts.googleapis.com/css?family=Lato:300,400,700,900" rel="stylesheet">
    <link rel="stylesheet" href="libs/font-awesome/css/font-awesome.min.css">
    <link href="css/bootstrap.min.css" rel="stylesheet">
    <link href="css/styles.css" rel="stylesheet">
</head>

<body>
    <div id="mobile-menu-open" class="shadow-large">
        <i class="fa fa-bars" aria-hidden="true"></i>
    </div>
    <!-- End #mobile-menu-toggle -->
    <header>
        <div id="mobile-menu-close">
            <span>Close</span> <i class="fa fa-times" aria-hidden="true"></i>
        </div>
        <ul id="menu" class="shadow">
            <li>
               <b> <a href="#about">About</a></b>
            </li>
            <li>
               <b> <a href="#experience">Experience</a></b>
            </li>
            <li>
               <b> <a href="#education">Education</a></b>
            </li>
            <li>
               <b> <a href="#projects">Projects</a></b>
            </li>
            <li>
               <b> <a href="#skills">Skills</a></b>
            </li>
            <li>
               <b> <a href="#publication">Publications</a></b>
            </li>			
            <li>
                <b> <a href="#contact">Contact</a></b>
            </li>
        </ul>
    </header>
    <!-- End header -->

    <div id="lead">
        <div id="lead-content">
            <h1>Ruijia(Ray) Wang</h1>
            <h2>Bioinformatician, Computational Biologist</h2>
            <a href="https://www.linkedin.com/in/ruijia-wang-85088546/" class="btn-rounded-white">Linkedin</a>
			<a href="https://github.com/RJWANGbioinfo" class="btn-rounded-white">GitHub</a>
			<a href="https://scholar.google.com/citations?user=OvXNbDwAAAAJ&hl=en" class="btn-rounded-white">Google Scholar</a>
			<a href="https://www.researchgate.net/profile/Ruijia_Wang" class="btn-rounded-white">ResearchGate</a>
        </div>
        <!-- End #lead-content -->

        <div id="lead-overlay"></div>

        <div id="lead-down">
            <span>
                <i class="fa fa-chevron-down" aria-hidden="true"></i>
            </span>
        </div>
        <!-- End #lead-down -->
    </div>
    <!-- End #lead -->

    <div id="about">
        <div class="container">
            <div class="row">
                <div class="col-md-4">
                    <h2 class="heading">About Me</h2>
					<!--<img class="image max-width-50" src="images/dummay-img.png">-->
                </div>
	
                <div class="col-md-8">
                    <p>
						<b>I'm a bioinformatician and computational biologist with Ph.D. degree in bioinformatics and genomics:</b>
						  <li> 5+ years’ experience in bioinformatics especially on analyzing large scale NGS data.</li>
						  <li> Proficient in bioinformatic programming/scripting with Python and R.</li>
						  <li> First/co-first/corresponding author of 20+ peer-reviewed research publication in high impact journals.</li>
						  <li> The developer of bioinformatics packages <b><a target="_blank" href="https://github.com/RJWANGbioinfo/APAlyzer">APAlyzer</a></b> and <b><a target="_blank" href="https://github.com/RJWANGbioinfo/APAlyzer-QSr">APAlyzer-QSr</a></b>.</li>
						  <li> Strong research background on genomics, transcriptomics, genetics and system biology. </li>
						  <li> Academic Editor of Scientific Reports.</li> 
			  </p>
                </div>
            </div>
        </div>
    </div>
    <!-- End #about -->

    <div id="experience" class="background-alt">
        <h2 class="heading">Experience</h2>
        <div id="experience-timeline">
            <div data-date="2019 – Present">
                <h3>Assistant Professor, Director of Bioinformatics</h3>
                <h4>University of Massachusetts Medical School</h4>
                <p>
			  <li> Lead the bioinformatics core of Program in Molecular Medicine</li>
			  <li> Provide bioinformatics-oriented collaborations and consultation on medical research</li>
			  <li> Bioinformatics pipeline development for the routine NGS analysis</li> 
			  <li> Bioinformatics package development on regulatory 3’end processing.</li>
			  <li> Manage and maintain the sequencing data using Google Cloud Storage Platform</li>
			  <li> Take charge of team development and recruitment of team members</li>
			  </p>
            </div>

            <div data-date="2016 – 2019">
                <h3>Postdoctoral Research Fellow</h3>
                <h4>Rutgers New Jersey Medical School</h4>
                <p>
			  <li> Construction of PolyA_DB3 database and evolution analysis on mammal PAS</li>
			  <li> Bioinformatic modeling on the transcriptional master regulator</li>
			  <li> APA based GWAS and eQTL in human population, cancer, and neurological disease</li>
			  <li> Collaborate with a UCSD team on APA regulation in acute myeloid leukemia patients</li>
              </p>
            </div>

            <div data-date="2014 – 2016">
                <h3>Associate Professor</h3>
                <h4>Ocean University of China</h4>
                <p>
                   <li> Lead the genomic resource development and whole genome project in echinoderm</li>
                </p>
            </div>
            <div data-date="2013 – 2014">
                <h3>Postdoctoral Research Fellow</h3>
                <h4>Auburn University</h4>
                <p>
                    <li> GWAS on the identification of marker genes on fish bacterial infectious diseases </li>
                </p>
            </div>			
			
			
        </div>
    </div>
    <!-- End #experience -->

    <div id="education">
        <h2 class="heading">Education</h2>
        <div class="education-block">
            <h3>Auburn University</h3>
            <span class="education-date">2009 - 2013</span>
            <h4>Ph.D., Genomics and Bioinformatics</h4>
        </div>
        <!-- End .education-block -->

        <div class="education-block">
            <h3>Ocean University of China</h3>
            <span class="education-date">2003 - 2007</span>
            <h4>Bachelor of Science in Biotechnology</h4>

        </div>
        <!-- End .education-block -->
    </div>
    <!-- End #education -->

    <div id="projects" class="background-alt">
        <h2 class="heading">Projects</h2>
        <div class="container">
            <div class="row">
                <div class="project shadow-large">
                    <div class="project-image">
                        <img src="images/Picture1.jpg" />
                    </div>
                    <!-- End .project-image -->
                    <div class="project-info">
                        <h3>Package development on alternative polyadenylation (APA) analysis using RNA-Seq data (2019 - Present) </h3>
                        <p align="justify">
						RNA-seq and 3’ end sequencing data have proven effective in revealing APA changes, however, 
						poor PAS annotations and intrinsic limitations of de novo PAS prediction are major challenges. 
						To overcome these, we developed <b>APAlyzer</b> and <b>APAlyzer-QSr</b>(Non-Exclusive Licenced to Admera Health) examine APA events in all genic regions
						using RNA-Seq and Quant-Seq REV, respectively.</p>
                        <a href="#">View Project</a>
                    </div>
                    <!-- End .project-info -->
                </div>
                <!-- End .project -->

                <div class="project shadow-large">
                    <div class="project-image">
                        <img src="images/Picture2.jpg" />
                    </div>
                    <!-- End .project-image -->
                    <div class="project-info">
                        <h3>Bioinformatic modeling on the regulation of alternative cleavage and polyadenylation (APA) (2016 - 2019)</h3>
                        <p align="justify">
						APA has been increasingly recognized as an important layer mRNA processing regulator which plays an important role in disease related process, 
						such as tumorigenesis and neurological disorder. In this ongoing research, we have studied export factor NXF1 and CPA factor PCF11, 
						revealing gene size-based and APA-based regulatory mechanisms. 
						We also collaborate with a UCSD team on APA regulation in acute myeloid leukemia 	                        
						</p>
                        <a href="project1.html">View Project</a>
                    </div>
                    <!-- End .project-info -->
                </div>
				
                <div class="project shadow-large">
                    <div class="project-image">
                        <img src="images/Picture3.jpg" />
                    </div>
                    <!-- End .project-image -->
                    <div class="project-info">
                        <h3>Construction of PolyA_DB3 database and genomic evolution analysis on mammal PAS (2016 - 2019)</h3>
                        <p align="justify">
						In this project, I constructed a vertebrate PAS database, 
						PolyA_DB 3. This database contains PAS identified in several vertebrates, 
						including humans, mouse, rat and chicken, drawn from over 100 datasets. 
						In addition, through systematic bioinformatics analysis, 
						we present the first comprehensive view of conserved cleavage and polyadenylation events in mammals. 
						
						</p>
                        <a href="#">View Project</a>
                    </div>
                    <!-- End .project-info -->
                </div>	
				
                <div class="project shadow-large">
                    <div class="project-image">
                        <img src="images/Picture4.jpg" />
                    </div>
                    <!-- End .project-image -->
                    <div class="project-info">
                        <h3>Genomic resource development in echinoderm <br>(2014 - 2016)</h3>
                        <p align="justify">
						In this project, my research mainly focusing on two aspects:<br>
						1) I lead the construction of the first high-density genetic linkage map, whole genome assembly and annotation for echinoderms in a sea cucumber species using the whole genome sequencing and 2b-RAD-Seq technique. 
						<br> 2) I analyzed transcriptome data from two echinoderm species and identified nearly 17,000 long-noncoding RNA (lncRNA), and connected the lncRNA, mRNA and miRNA as the lncRNA-microRNA-gene trinities to construct the first coding-noncoding network in echinoderm. 
						
						</p>
                        <a href="#">View Project</a>
                    </div>
                    <!-- End .project-info -->
                </div>

                <div class="project shadow-large">
                    <div class="project-image">
                        <img src="images/Picture5.jpg" />
                    </div>
                    <!-- End .project-image -->
                    <div class="project-info">
                        <h3>Identification of marker genes relevant to bacterial infectious diseases in fish (2012 - 2014)</h3>
                        <p align="justify">
						I optimized Bulk segregant RNA-sequencing (BSR-seq) technology and applied it to study enteric septicemia of catfish, 
						one of the most common and serious diseases afflicting catfish. 
						This study demonstrated the utility and effectiveness of this novel technique and identified the candidate genomic region relevant to this disease. We also performed a series of genomic and bioinformatics analyses of the data obtained using BSR-seq, 
						making a range of crucial discoveries concerning catfish innate immunity and resistance against bacterial infections.					
						</p>
                        <a href="#">View Project</a>
                    </div>
                    <!-- End .project-info -->
                </div>	
				
                <!-- End .project -->
            </div>
        </div>
    </div>
    <!-- End #projects -->

    <div id="skills">
        <h2 class="heading">Skills</h2>
        <ul>
            <li>Bioinformatics </li>			
            <li>Python</li>
            <li>R</li>
			<li>Pipeline development</li>
			<li>Package development</li>
            <li>HPCC</li>
            <li>Linux/UNIX </li>
            <li>Expression, splicing, and Alternative Polyadenylation profiling</li>
            <li>bulk RNA-seq</li>
            <li>single-cell RNA-seq</li>
            <li>ChIP-seq</li>
            <li>NET-seq</li>
            <li>iCLIP</li>
            <li>CutRun</li>
            <li>ATAC-seq</li>
            <li>SNP</li>
            <li>PAS</li>
            <li>Motif analysis </li>
            <li>SNP calling </li>
            <li>GWAS</li>
            <li>QTL and eQTL</li>
            <li>Genomic resource development</li>
            <li>Database construction</li>
            <li>De novo whole genome assembly</li>
            <li>Whole genome DNA-Seq</li>
			<li>High-density linkage map construction</li>	
			<li>LncRNA identification</li>
			<li>Coding-non-coding co-expression network</li>
			<li>ENCODE</li>
			<li>GEO</li>
			<li>Uniprot</li>
			<li>DisGeNET</li>
			<li>TCGA</li>
			<li>GTEx</li>
			<li>dbGaP</li>
			<li>dbSNP</li>
			<li>GO</li>
			<li>KEGG</li>
			<li>BLAST</li>
			<li>IPA</li>
			<li>CLC</li>
			<li>Cytoscape</li>
			<li>UCSC Genome Browser</li>
			<li>IGV</li>
			<li>samtools</li>
			<li>GATK</li>
			<li>bedtools</li>
			<li>VarScan</li>
			<li>Bowtie</li>
			<li>STAR</li>
			<li>rMATS</li>
			<li>cellranger</li>
			<li>PLINK</li>
			<li>seurat</li>			
        </ul>
    </div>
    <!-- End #skills -->

    <div id="publication" class="background-alt">
        <h2 class="heading">Publications</h2>
			<div class="container">
			
				<div class="row">
					<!--<img class="image max-width-50" src="images/dummay-img.png">-->
                    <p class="text">
						<b>* Equal contribution. <u>Name underlined</u> when as corresponding author.</b><br>
						<b>Complete List of Publication in <a target="_blank" href="https://scholar.google.com/citations?user=OvXNbDwAAAAJ&hl=en" >Google Scholar</a></b>
					</p>
				</div>			
			
				<div class="row">
					<!--<img class="image max-width-50" src="images/dummay-img.png">-->
                    <p class="text">
						<b>2020:</b><br>
						<ol>
						<li><b>Ruijia Wang</b> and Bin Tian: APAlyzer: a bioinformatic package for analysis of alternative polyadenylation isoforms. Bioinformatics (2020): (Under review).</li>	
						<li>Shah, Sneha, Gemma Molinaro, Botao Liu, <b>Ruijia Wang</b>, Kimberly M. Huber, and Joel D. Richter: FMRP Control of Ribosome Translocation Promotes Chromatin Modifications and Alternative Splicing of Neuronal Genes Linked to Autism. Cell Reports (2020): (Under review).</li>	
						</ol>
					</p>
				</div>
				
				<div class="row">
					<!--<img class="image max-width-50" src="images/dummay-img.png">-->
                    <p class="text">
						<b>2019:</b><br>
						<ol>
						<li><b>Ruijia Wang</b>, Dinghai Zheng, Lu Wei, Qingbao Ding, Bin Tian: Regulation of intronic polyadenylation by PCF11 impacts mRNA expression of long genes. Cell Reports, 26.10 (2019):2766-2778.e2766.</li>
						<li>Suli Chen*, <b>Ruijia Wang*</b>, Dinghai Zheng*, Heng Zhang*, Xingya Chang, Ke Wang, Wencheng Li, Jing Fan, Bin Tian, Hong Cheng: The mRNA export receptor NXF1 coordinates transcriptional dynamics, alternative polyadenylation and mRNA export. Molecular Cell, 1.74 (2019):118-131.</li>
						</ol>
					</p>
				</div>

				<div class="row">
					<!--<img class="image max-width-50" src="images/dummay-img.png">-->
                    <p class="text">
						<b>2018:</b><br>
						<ol>
						<li><b>Ruijia Wang</b>, Dinghai Zheng, Ghassan Yehia, Bin Tian: A compendium of conserved cleavage and polyadenylation events in mammalian genes. Genome research 28.10 (2018): 1427-1441.</li>
						<li><b>Ruijia Wang</b>, Ram Nambiar, Dinghai Zheng, Bin Tian: PolyA_DB 3 catalogs cleavage and polyadenylation sites identified by deep sequencing in multiple genomes. Nucleic acids research 46, no. D1 (2017): D315-D319.</li>
						<li>Dinghai Zheng*, <b>Ruijia Wang*</b>, Qingbao Ding, Tianying Wang, Bingning Xie, Lu Wei, Zhaohua Zhong, Bin Tian: Cellular stress alters 3’ UTR landscape through alternative polyadenylation and isoform-specific degradation. Nature communications 9.1 (2018): 2268.</li>
						<li>Yuli Li*, <b>Ruijia Wang*</b>, Xiaogang Xun, Jing Wang, Lisui Bao, Ramesha Thimmappa, Jun Ding, Jingwei Jiang, Liheng Zhang, Tianqi Li, Jia Lv, Chuang Mu, Xiaoli Hu, Lingling Zhang, Jing Liu, Yuqiang Li, Lijie Yao, Wenqian Jiao, Yangfan Wang, Shanshan Lian, Zelong Zhao, Yaoyao Zhan, Xiaoting Huang, Huan Liao, Jia Wang, Hongzhen Sun, Xue Mi, Yu Xia, Qiang Xing, Wei Lu, Anne Osbourn, Zunchun Zhou, Yaqing Chang, Zhenmin Bao, Shi Wang: Sea cucumber genome provides insights into saponin biosynthesis and aestivation regulation. Cell discovery 4.1 (2018): 29.</li>
						<li>Jia Lv, Wenqian Jiao, Haobing Guo, Pingping Liu, <b>Ruijia Wang</b>, Lingling Zhang, Qifan Zeng, Xiaoli Hu, Zhenmin Bao, Shi Wang: HD-Marker: a highly multiplexed and flexible approach for targeted genotyping of more than 10,000 genes in a single-tube assay. Genome research, 28.12 (2018):  1919-1930.</li>
						</ol>
					</p>
				</div>	

				<div class="row">
					<!--<img class="image max-width-50" src="images/dummay-img.png">-->
                    <p class="text">
						<b>2017:</b><br>
						<ol>
						<li><b>Ruijia Wang</b>, Lisui Bao, Shikai Liu and Zhanjiang Liu: Chapter 4. Genome Annotation: Determination of the Coding Potential of the genome. Bioinformatics in Aquaculture: Principles and Methods; John Wiley & Sons, Inc. 2017, 74</li>
						<li><b>Ruijia Wang</b> and Zhanjiang Liu: Chapter 6. Analysis of duplicated genes and multi-gene families. Bioinformatics in Aquaculture: Principles and Methods; John Wiley & Sons, Inc. 2017, 98</li>
						<li><b>Ruijia Wang</b>, Lisui Bao, Shikai Liu and Zhanjiang Liu: Chapter 12. Analysis of long non-coding RNAs (lncRNAs). Bioinformatics in Aquaculture: Principles and Methods; John Wiley & Sons, Inc. 2017, 179-199.</li>
						<li>Jun Yao, <b>Ruijia Wang</b> and Zhanjiang Liu: Chapter 11. Genetic analysis using RNA-Seq: Bulk segregant RNA-Seq (BSR-Seq). Bioinformatics in Aquaculture: Principles and Methods; John Wiley & Sons, Inc. 2017, 169-178</li>
						<li>Yun Li, Shikai Liu, <b>Ruijia Wang</b>, Zhenkui Qin and Zhanjiang Liu: Chapter 20. Bioinformatic considerations and approaches for high-density Linkage Mapping in Aquaculture. Bioinformatics in Aquaculture: Principles and Methods; John Wiley & Sons, Inc. 2017, 356-379</li>
						<li>Shi Wang, Jinbo Zhang, Wenqian Jiao, Ji Li, Xiaogang Xun, Yan Sun, Ximing Guo, Pin Huan, Bo Dong, Lingling Zhang, Xiaoli Hu, Xiaoqing Sun, Jing Wang, Chengtian Zhao, Yangfan Wang, Dawei Wang, Xiaoting Huang, <b>Ruijia Wang</b>, Jia Lv, Yuli Li, Zhifeng Zhang, Baozhong Liu, Wei Lu, Yuanyuan Hui, Jun Liang, Zunchun Zhou, Rui Hou, Xue Li, Yunchao Liu, Hengde Li, Xianhui Ning, Yu Lin, Liang Zhao, Qiang Xing, Jinzhuang Dou, Yangping Li, Junxia Mao, Haobing Guo, Huaiqian Dou, Tianqi Li, Chuang Mu, Wenkai Jiang, Qiang Fu, Xiaoteng Fu, Yan Miao, Jian Liu, Qian Yu, Ruojiao Li, Huan Liao, Xuan Li, Yifan Kong, Zhi Jiang, Daniel Chourrout, Ruiqiang Li & Zhenmin Bao:    Scallop genome provides insights into evolution of bilaterian karyotype and development  Nature ecology & evolution 1.5 (2017): 0120.</li>
						</ol>
					</p>
				</div>	
				
				<div class="row">
					<!--<img class="image max-width-50" src="images/dummay-img.png">-->
                    <p class="text">
						<b>Selected publication from 2012-2016:</b><br>
						<ol>
						<li><b>Ruijia Wang</b>, Luyang Sun, Lisui Bao, Jiaren Zhang, Yanliang Jiang, Jun Yao, Lin Song, Jianbin Feng, Shikai Liu, Zhanjiang Liu: Bulk segregant RNA-seq reveals expression and positional candidate genes and allele-specific expression for disease resistance against enteric septicemia of catfish. BMC Genomics (2013); 14(1):929. </li>
						<li><b>Ruijia Wang</b>, Yu Zhang, Shikai Liu, Chao Li, Luyang Sun, Lisui Bao, Jianbin Feng, Zhanjiang Liu: Analysis of 52 Rab GTPases from channel catfish and their involvement in immune responses after bacterial infections. Developmental and comparative immunology 45.1 (2014): 21-34. </li>
						<li><b>Ruijia Wang</b>, Jianbin Feng, Chao Li, Shikai Liu, Yu Zhang, Zhanjiang Liu: Four lysozymes (one c-type and three g-type) in catfish are drastically but differentially induced after bacterial infection. Fish & Shellfish Immunology (2013); 35(1). </li>
						<li><b>Ruijia Wang</b>, Chao Li, James Stoeckel, Gregory Moyer, Zhanjiang Liu, Eric Peatman: Rapid development of molecular resources for a freshwater mussel, Villosa lienosa (Bivalvia:Unionidae), using an RNA-seq- based approach. Freshwater science (2012); 31(3):695-708.</li>
						<li>Xue Li, <u><b>Ruijia Wang</b></u>, Xiaogang Xun, Wenqian Jiao, Mengran Zhang, Shuyue Wang, Shi Wang, Lingling Zhang, Xiaoting Huang, Xiaoli Hu, Zhenmin Bao: The Rho GTPase Family Genes in Bivalvia Genomes: Sequence, Evolution and Expression Analysis. PloS one 10.12 (2015): e0143932.</li>
						<li>Chuang Mu*, <u><b>Ruijia Wang*</b></u>, Tianqi Li*, Yuqiang Li, Meilin Tian, Wenqian Jiao, Xiaoting Huang, Lingling Zhang, Xiaoli Hu, Shi Wang, Zhenmin Bao: Long non-coding RNAs (lncRNAs) in sea cucumber: The large- scale identification, expression profiling, coding-non-coding network construction. Marine biotechnology 18.4 (2016): 485-499.</li>
						<li>Ning, Xianhui, <u><b>Ruijia Wang</b></u>, Xue Li, Shuyue Wang, Mengran Zhang, Qiang Xing, Yan Sun, Shi Wang, Lingling Zhang, Xiaoli Hu, Zhenmin Bao: Genome-wide identification and characterization of five MyD88 duplication genes in Yesso scallop (Patinopecten yessoensis) and expression changes in response to bacterial challenge. Fish & shellfish immunology 46, no. 2 (2015): 181-191.</li> 
						<li>Jing Wang, <u><b>Ruijia Wang</b></u>, Shuyue Wang, Mengran Zhang, Xiaoli Ma, Pingping Liu, Meiwei Zhang, Xiaoli Hu, Lingling Zhang, Shi Wang, Zhenmin Bao: Genome-wide identification and characterization of TRAF genes in the Yesso scallop (Patinopecten yessoensis) and their distinct expression patterns in response to bacterial challenge. Fish & Shellfish Immunology (2015), doi:10.1016/j.fsi.2015.09.050. </li>
						<li>Zou, Jiajun, <u><b>Ruijia Wang</b></u>, Ruojiao Li, Yifan Kong, Jing Wang, Xianhui Ning, Lingling Zhang, Shi Wang, Xiaoli Hu, and Zhenmin Bao: The genome-wide identification of mitogen-activated protein kinase kinase (MKK) genes in Yesso scallop Patinopecten yessoensis and their expression responses to bacteria challenges. Fish & shellfish immunology 45, no. 2 (2015): 901-911. </li>
						<li>Meilin Tian, Yangping Li, Jing Jing, Chuang Mu, Huixia Du, Jinzhuang Dou, Junxia Mao, Xue Li, Wenqian Jiao, Yangfan Wang, Xiaoli Hu, Shi Wang, <u><b>Ruijia Wang</b></u>, Zhenmin Bao: Construction of a High-Density Genetic Map and Quantitative Trait Locus Mapping in the Sea Cucumber Apostichopus japonicus. Scientific Reports 5 (2015), 14852, doi:10.1038/srep14852.</li>
						<li>Chao Li, <b>Ruijia Wang</b>, Baofeng Su, Yupeng Luo, Jeffery Terhune, Benjamin Beck, Eric Peatman: Evasion of Mucosal Defenses During Aeromonas hydrophila Infection of Channel Catfish (Ictalurus punctatus) Skin. Developmental and comparative immunology 39.4 (2013): 447-455. </li>
						<li>Chao Li, Yu Zhang, <b>Ruijia Wang</b>, Jianguo Lu, Samiran Nandi, Sriprakash Mohanty, Jeffery Terhune, Zhanjiang Liu, Eric Peatman: RNA-seq analysis of mucosal immune responses reveals signatures of intestinal barrier disruption and pathogen entry following Edwardsiella ictaluri infection in channel catfish, Ictalurus punctatus. Fish & Shellfish Immunology 32.5 (2012):816-27.</li>
						<li>Ailu Chen, <b>Ruijia Wang</b>, Shikai Liu, Eric Peatman, Luyang Sun, Lisui Bao, Chen Jiang, Chao Li, Yun Li, Qifan Zeng, Zhanjiang Liu: Ribosomal protein genes are highly enriched among genes with allele-specific expression in the interspecific F1 hybrid catfish. Molecular Genetics and Genomics, 291.3 (2016): 1083-1093. </li>
						<li>Zhanjiang Liu, Shikai Liu, Jun Yao, Lisui Bao, Jiaren Zhang, Yun Li, Chen Jiang, Luyang Sun, <b>Ruijia Wang</b>, Yu Zhang, Tao Zhou, Qifan Zeng, Qiang Fu, Sen Gao, Ning Li, Sergey Koren, Yanliang Jiang, Aleksey Zimin, Peng Xu, Adam M. Phillippy, Xin Geng, Lin Song, Fanyue Sun, Chao Li, Xiaozhu Wang, Ailu Chen, Yulin Jin, Zihao Yuan, Yujia Yang, Suxu Tan, Eric Peatman, Jianguo Lu, Zhenkui Qin, Rex Dunham, Zhaoxia Li, Tad Sonstegard, Jianbin Feng, Roy G. Danzmann, Steven Schroeder, Brian Scheffler, Mary V. Duke, Linda Ballard, Huseyin Kucuktas, Ludmilla Kaltenboeck, Haixia Liu, Jonathan Armbruster, Yangjie Xie, Mona L. Kirby, Yi Tian, Mary Elizabeth Flanagan, Weijie Mu, Geoffrey C. Waldbieser:    The channel catfish genome sequence provides insights into the evolution of scale formation in teleosts. Nature communications 7 (2016): 11757.</li>
						</ol>
					</p>
				</div>					
				
			</div>
    </div>
    <!-- End #about -->
	
	
    <div id="contact">
        <h2>Get in Touch</h2>
        <div id="contact-form">
            <form method="POST" action="https://formspree.io/email@email.com">
                <input type="hidden" name="_subject" value="Contact request from personal website" />
                <input type="email" name="_replyto" placeholder="Your email" required>
                <textarea name="message" placeholder="Your message" required></textarea>
                <button type="submit">Send</button>
            </form>
        </div>
        <!-- End #contact-form -->
    </div>
    <!-- End #contact -->

    <footer>
        <div class="container">
            <div class="row">
                <div class="col-sm-5 copyright">
                    <p>
                        Copyright &copy; 2020 Ruijia Wang
                    </p>
                </div>
                <div class="col-sm-2 top">
                    <span id="to-top">
                        <i class="fa fa-chevron-up" aria-hidden="true"></i>
                    </span>
                </div>
                <div class="col-sm-5 social">
                    <ul>
                        <li>
                            <a href="https://github.com/" target="_blank"><i class="fa fa-github" aria-hidden="true"></i></a>
                        </li>
                        <li>
                            <a href="https://stackoverflow.com/" target="_blank"><i class="fa fa-stack-overflow" aria-hidden="true"></i></a>
                        </li>
                        <li>
                            <a href="https://linkedin.com/" target="_blank"><i class="fa fa-linkedin" aria-hidden="true"></i></a>
                        </li>
                        <li>
                            <a href="https://www.facebook.com/" target="_blank"><i class="fa fa-facebook" aria-hidden="true"></i></a>
                        </li>
                        <li>
                            <a href="https://twitter.com/" target="_blank"><i class="fa fa-twitter" aria-hidden="true"></i></a>
                        </li>

                    </ul>
                </div>
            </div>
        </div>
    </footer>
    <!-- End footer -->

    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
    <script src="js/scripts.min.js"></script>
</body>

</html>
