<!doctype html>
<html>

<head>
  <title>PolyA_DB3 and evolution of PAS</title>
  <meta charset="utf-8" name="viewport" content="width=device-width, initial-scale=1">
  <link href="css/frame.css" media="screen" rel="stylesheet" type="text/css" />
  <link href="css/controls.css" media="screen" rel="stylesheet" type="text/css" />
  <link href="css/custom.css" media="screen" rel="stylesheet" type="text/css" />
  <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,700' rel='stylesheet' type='text/css'>
  <link href='https://fonts.googleapis.com/css?family=Open+Sans+Condensed:300,700' rel='stylesheet' type='text/css'>
  <link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro:400,700" rel="stylesheet">
  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
  <style>
    .menu-index {
      color: rgb(255, 255, 255) !important;
      opacity: 1 !important;
      font-weight: 700 !important;
    }
  </style>
</head>
<div class="menu-container">
<div class="menu">
  <div class="menu-table flex-row-space-between">
    <div class="logo flex-row-center">
      <a href="#">PolyA_DB3 and evolution of PAS</a>
    </div>
    <a class="menu-button" tabindex="0" href="javascript:void(0)">
      <img src="img/menu.png">
    </a>
    <div class="menu-items flex-row-center flex-item">
      <a href="#" class="menu-index">Overview</a>
	  <a href="http://exon.umdnj.edu/polya_db/v3/" class="menu-index">PolyA_DB 3</a>
	  <a href="index.html" class="menu-index">My website</a>
    </div>
  </div>
</div>
</div>
<body>
  <div class="menu-container"></div>
  <div class="content-container">
    <div class="content">
      <div class="content-table flex-column">
        <!--Start Intro-->


        <!--End Intro-->
        <!--Start Education-->
        <div class="flex-row">

        <div class="flex-row">
          <div class="flex-item flex-column">
            <h2 class="add-top-margin">Project Overview</h2>
            <hr>
            <p class="text" align="justify">
              <img class="image image-wrap-text max-width-400" src="images/p3_n1.png">
			In this project, I constructed a vertebrate PAS database, <b><a target="_blank" href="http://exon.umdnj.edu/polya_db/v3/">PolyA_DB 3</a></b>. 
			This database contains PAS identified in several vertebrates, including humans, mouse, rat and chicken, drawn from over 100 datasets. 
			In additions, through systematic bioinformatics analysis, we present the first comprehensive view of conserved cleavage and polyadenylation events in mammals. 
            </p>
          </div>
        </div>		
          <div class="flex-item flex-column">
            <h2 class="add-top-margin">Representative publication</h2>
            <hr>
            <p class="text">			
						<ol>
						<li><b>Ruijia Wang</b>, Dinghai Zheng, Ghassan Yehia, Bin Tian: A compendium of conserved cleavage and polyadenylation events in mammalian genes. Genome research 28.10 (2018): 1427-1441. <b><a target="_blank" href="https://genome.cshlp.org/content/28/10/1427.full">Paper link</a></b></li>
						<li><b>Ruijia Wang</b>, Ram Nambiar, Dinghai Zheng, Bin Tian: PolyA_DB 3 catalogs cleavage and polyadenylation sites identified by deep sequencing in multiple genomes. Nucleic acids research 46, no. D1 (2017): D315-D319. <b><a target="_blank" href="https://academic.oup.com/nar/article/46/D1/D315/4561640">Paper link</a></b></li>
						</ol>

            </p>			
        </div>		
        <!--Start Text with Centered Image-->
        <div class="flex-row">
          <div class="flex-item flex-column">
            <h2 class="add-top-margin">Gallery</h2>
            <hr>

            <img class="image center" src="images/p3_g1.png">
            <p class="text"  align="justify">
			<b>Figure. PolyA_DB 3 construction and genomic evolution analysis on mammal PAS.</b><br>
			<b>(A)</b>  Schematic of PAS identification and presentation in PolyA_DB version 3.
			<b>(B)</b> Correlation of aUTR length (left) or cUTR length (right) between human and mouse genes. 
						A total of 6706 orthologous genes with conserved 3′ UTR APA are included. 
						3′ UTRs are divided into aUTRs and cUTRs using the first conserved and last conserved PASs (mammal conserved). 
						Median length value for each species and Pearson correlation coefficient are indicated. 
														<b>(C)</b> Sequence motifs at the cleavage site, shown as sequence logos. 
														Two scenarios are considered, as illustrated above the logos. 
														(Left) Cleavage site is considered to be immediately after a non-A nucleoside (shown as B) and before an adenosine. 
														(Right) Cleavage site is considered to be immediately after an adenosine and before a B.
														<b>(D)</b> Tetramers enriched for 3’UTRs of stable or unstable transcripts.
														<b>(E)</b> Odds ratio of miRNA target site distribution between conserved and nonconserved aUTRs for highly conserved miRNA families and moderately conserved miRNA families. miRNA
																	conservation is defined by TargetScan (“Family Conservation” score = 1 for moderately conserved miRNAs, and = 2 for highly conserved ones).

            </p>
          </div>
        </div>
        <!--End Text with Centered Image-->

        <!--Start Credits-->
        <div class="flex-row">
          <div class="flex-item flex-item-stretch flex-column">
            <p class="text text-small text-italic">
              Author: <span class="highlight-text">Ruijia Wang</span>
            </p>
          </div>
        </div>
        <!--End Credits-->
      </div>
    </div>
  </div>
</body>

</html>